test/input/json/RnR_test.json

```diff
@@ -19,6 +19,12 @@
     "driver_string": "NetCDF",
     "layer_string": 0,
     "waterbody_parameter_file_type": "Level_Pool",
+    "ql_input_folder": "/home/APD/inland_hydraulics/wrf-hydro-run/OUTPUTS",
+    "ql_files_tail": "/*.CHRTOUT_DOMAIN1",
+    "time_string": "2020-03-19_18:00_DOMAIN1",
+    "channel_initial_states_file": "/home/APD/inland_hydraulics/wrf-hydro-run/restart/HYDRO_RST.",
+    "initial_states_waterbody_ID_crosswalk_file": "/home/APD/inland_hydraulics/wrf-hydro-run/DOMAIN/waterbody_subset_ID_crosswalk.csv",
```

Comment:
Need to add the streams crosswalk file. (We tested with a default, but we'll want to call it out explicitly...)

Reply:
The streams crosswalk file being  "channel_initial_states_file": "/home/APD/inland_hydraulics/wrf-hydro-run/restart/HYDRO_RST.2020-03-19_18:00_DOMAIN1" ? Should it be combined or am I pulling in a different streams crosswalk file beyond the one being used here? Just looking for clarification here.

Reply:
For now, we'll use one of the ql files. Any one of them will do.